packages/graph/.env

```diff
@@ -1,17 +1,11 @@
-HASH_GRAPH_PG_USER=postgres
-HASH_GRAPH_PG_PASSWORD=postgres
-HASH_GRAPH_PG_HOST=localhost
-HASH_GRAPH_PG_PORT=5432
-HASH_GRAPH_PG_DATABASE=graph
+POSTGRES_PORT=5432
+POSTGRES_HOST=localhost
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
+POSTGRES_DB=postgres
 
-HASH_GRAPH_PG_MIGRATION_URL=postgres://${HASH_GRAPH_PG_USER}:${HASH_GRAPH_PG_PASSWORD}@${HASH_GRAPH_PG_HOST}:${HASH_GRAPH_PG_PORT}/${HASH_GRAPH_PG_DATABASE}
+HASH_GRAPH_PG_USER=graph
+HASH_GRAPH_PG_PASSWORD=graph
+HASH_GRAPH_PG_DATABASE=graph
 
-###########################################
-## Aliases for dockerized external services
-###########################################
-##
-## These variables are consumed in deployment/docker-compose.yml → env_file.
-## Using `environment` → POSTGRES_USER=${HASH_GRAPH_PG_USER} does not give access to values in this file.
```

Comment:
I've removed this comment as these environment variables no longer server purely as aliases